modules/apps/polls/polls-web/src/main/resources/META-INF/resources/polls/view_question.jsp

```diff
@@ -60,19 +62,19 @@ portletDisplay.setURLBack(redirect);
 	<aui:fieldset-group markupView="lexicon">
 		<aui:fieldset>
 			<h2>
-				<%= StringUtil.replace(question.getDescription(locale), CharPool.NEW_LINE, "<br />") %>
+				<%= StringUtil.replace(questionDescription.toString(), CharPool.NEW_LINE, "<br />") %>
 			</h2>
 
 			<c:choose>
 				<c:when test="<%= !viewResults && !question.isExpired() && !hasVoted && PollsQuestionPermission.contains(permissionChecker, question, ActionKeys.ADD_VOTE) %>">
 
 					<%
 					for (PollsChoice choice : choices) {
-						choice = choice.toEscapedModel();
+						UserInputString choiceDescription = choice.getDescription(locale);
 					%>
 
 						<aui:field-wrapper cssClass="radio">
-							<aui:input label='<%= choice.getName() + ". " + choice.getDescription(locale) %>' name="choiceId" type="radio" value="<%= choice.getChoiceId() %>" />
+							<aui:input label='<%= choice.getName() + ". " + choiceDescription.unsafeGetString() %>' name="choiceId" type="radio" value="<%= choice.getChoiceId() %>" />
```

Comment:
we should not use `unsafeGetString` in this one... we should use `map` but it looks we can't use lambdas in the JSP :-(